komodod/importprivkey

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+chain=$ASSET_NAME
+komodo-cli -ac_name=$chain importprivkey Ut1Xx8tAePPxJgTkEVAwj3ZhRvBZ7gY5gJ5ob4QAPiCDcyMJEYgH
```

Comment:
Please pass privkey as ENV variable. Again, we do not want to rebuild images in case this would change.